package-lock.json

```diff
@@ -1861,8 +1861,8 @@
       }
     },
     "node_modules/@dynamods/notifications-panel": {
-      "version": "0.0.6",
-      "resolved": "https://registry.npmjs.org/@dynamods/notifications-panel/-/notifications-panel-0.0.6.tgz",
+      "version": "0.0.7",
+      "resolved": "https://registry.npmjs.org/@dynamods/notifications-panel/-/notifications-panel-0.0.7.tgz",
```

Comment:
This would cause the sha512 mismatch with the version

Reply:
@QilongTang I've rolled back my changes according to your suggestion but the build still failed, unfortunately

Reply:
@QilongTang  @zeusongit Just for the record 
`npm i -g npm
rm -rf node_modules/
npm cache clear --force 
npm i`
solved the checksum integrity issue